phs.ttl

```diff
@@ -190,6 +219,23 @@
                       skos:prefLabel "Diagnostic Method Type"@en .
 
 
+###  http://linked.data.gov.au/def/phs#Bycatch
+:Bycatch rdf:type owl:Class ;
```

Comment:
properties/linkage of bycatch with the ability to become a sample needs to be supported